pkg/ebpf/telemetry/errors_telemetry.go

```diff
@@ -40,7 +43,22 @@ var helperNames = map[int]string{
 	perfEventOutput: "bpf_perf_event_output",
 }
 
-// EBPFTelemetry struct contains all the maps that
+type telemetryIndex struct {
+	key  uint64
+	name string
+}
+
+type ebpfErrorsTelemetry interface {
```

Comment:
what is the function of the interface? The only exported functions are `Lock` and `Unlock` and I think those probably shouldn't be exported anyways.

Reply:
this allows simple UTs (using mocks), added a description in code